CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Adjust `Alert` component for `CMS` ([#29] (https://github.com/vtex-sites/gatsby.store/pull/29))
```

Comment:
This extra space breaks the link formatting:
```suggestion
- Adjust `Alert` component for `CMS` ([#29](https://github.com/vtex-sites/gatsby.store/pull/29))
```

Reply:
Good catch!